JS_HW_3/HW_3.js

```diff
@@ -0,0 +1,43 @@
+//Задача 1
+
+console.log(addThemAll(2,4)); // 6
+console.log(addThemAll(1,2,3,4)); // 10
+console.log(addThemAll(5,5,10)); // 20
+
+function addThemAll(...args) {
+    let arr = [...args];
+    let sum = 0;
+    for(let item of arr) {
```

Comment:
```suggestion
    let sum = 0;
    for(let item of args) {
```
Проміжний масив не потрібен, ми можемо зразу ітерувати через args